examples/open-embed-webcontainer/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>StackBlitz SDK - Open and embed a StackBlitz WebContainer project</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <script type="module" src="./index.ts"></script>
+  </head>
+  <body>
+    <header>
+      <h1>Open and embed a StackBlitz WebContainer project</h1>
+      <div>
+        <label>
+          <input type="checkbox" name="corp" /> Cross-Origin Isolation
```

Comment:
Checking and unchecking this checkbox will load the page either with or without the CORP headers making it possible to test how embeds behave in different scenario's.